paddle/scripts/paddle_build.sh

```diff
@@ -487,7 +487,17 @@ function assert_api_spec_approvals() {
         BRANCH="develop"
     fi
 
-    API_FILES=("paddle/fluid/API.spec" "paddle/fluid/framework/operator.h")
+    API_FILES=("paddle/fluid/API.spec"
+               "paddle/fluid/framework/operator.h"
+               "paddle/fluid/framework/tensor.h"
+               "paddle/fluid/framework/lod_tensor.h"
+               "paddle/fluid/framework/selected_rows.h"
+               "paddle/fluid/framework/op_desc.h"
```

Comment:
how about adding `framework.proto` to replace `xx_desc.h`?

Reply:
also `send_recv.proto.in`

Reply:
Added 2 more files. And keep xx_desc.h since they are also very important.